### DISCUSSION TITLE
A highlight problem about TOC

### DISCUSSION THREAD

**Discussion by linxianmiao** [squidfunk/mkdocs-material#3015] [Q&A] [answered]
I clone your repo and start the project the same as [mkdocs-material](https://squidfunk.github.io/mkdocs-material/creating-your-site/#building-your-site), but when the page scrolls to the bottom, the bottom of the TOC is not highlighted. But I see the TOC works correct in your site.
 I don't know why.Because the code is absolutely same.
I feel sorry that my English is not so good.

**Comment by linxianmiao** [comment, marked as answer]
By the way , I see different TOC performance in these website, [google developer](https://developers.google.com/apps-script/guides/html/best-practices) and [stripe](https://stripe.com/docs/revenue-recognition/data-import#additional-verifications) .Their TOC has some problem when the page scrolls to the bottom or click the item in TOC.

**Comment by squidfunk** [comment]
I'm not sure I understand what you mean by "highlight" - could you maybe provide a screenshot to make it clearer?

**Reply by linxianmiao** [reply]
![mkdocs-material](https://user-images.githubusercontent.com/83006176/132934493-c90ae7d6-b75f-45c5-9ab3-df41cc6966e9.png)
![google](https://user-images.githubusercontent.com/83006176/132934496-2ddea299-37e6-4f39-bf4a-09b9f306c057.png)

**Reply by linxianmiao** [reply]
and I clone your code and start a same projecrt, but the TOC is different. Let's see the next screenshot.
![my-project](https://user-images.githubusercontent.com/83006176/132934599-9b40600d-a8b8-4af7-8a90-f29c882c0828.png)

**Reply by linxianmiao** [reply]
I read your source code in `/src/assets/javascripts/components/toc/index.ts`, but didn't find the reason.

**Reply by squidfunk** [reply]
Thanks for the screenshots! Yes, you're right, the behavior is slightly different because the official documentation is built with [Insiders](https://squidfunk.github.io/mkdocs-material/insiders/) which has this behavior. It was added due to the [anchor tracking](https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#anchor-tracking) feature, so that when you're at the bottom of a page and copy the URL, the person clicking on that URL will also be guaranteed to be at the bottom of the page.

When the funding goal for this feature is reached, the behavior will be merged back into the community edition.

**Reply by linxianmiao** [reply]
Hahaha，I'm sorry that I don't see your reply because of the "View more" button.
